lib/build.py

```diff
@@ -321,6 +321,24 @@ def env_env(self):
       else:
          return self.image.metadata["env"]
 
+   @property
+   def label_arg(self):
+      if (self.image is None):
+         assert False, "unimplemented" # return dict()
+      else:
+         return self.image.metadata["arg"]
+
+   @property
+   def label_build(self):
+      return { **self.label_arg, **self.label_label }
+
+   @property
+   def label_label(self):
+      if (self.image is None):
+         assert False, "unimplemented" # return dict()
+      else:
+         return self.image.metadata["labels"]
+
```

Comment:
I think these are not needed, for two reasons: (1) there is no distinction between “labels to save in the container’s environment”, akin to `ENV`, and “labels to use in the build environment”, i.e., the union of `ARG` and `ENV`, and (2) because we don't actually *use* the labels for anything, I don't think we need any property for access within Charliecloud.